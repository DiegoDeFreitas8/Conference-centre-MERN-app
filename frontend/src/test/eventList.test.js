//imported my component and renderer
import React from "react";
import EventList from "../components/eventList"
import renderer from 'react-test-renderer';

//created a snapshot test of my component 
//that i have imported
test('renders correctly', () => {
    const tree = renderer
        .create(<EventList/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
})