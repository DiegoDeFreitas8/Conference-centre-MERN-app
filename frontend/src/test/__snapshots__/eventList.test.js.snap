// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="events-container"
>
  <button
    className="Button btn btn-primary"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    Display Events
  </button>
  <br />
  <br />
  <button
    className="Button btn btn-primary"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    Logout
  </button>
  <br />
  <br />
   
   
  <br />
  <br />
</div>
`;
